app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    count_friends = 0
+    count_masks = 0
+    for friend in friends:
+        try:
+            Cafe.visit_cafe(cafe, visitor=friend)
+            count_friends += 1
+        except NotWearingMaskError:
+            count_masks += 1
+        except (OutdatedVaccineError, NotVaccinatedError):
```

Comment:
OutdatedVaccineError, NotVaccinatedError
As for you can replace in by one error
Why you create this errors by Inheritance?)